examples/example.env

```
@@ -0,0 +1,12 @@
+# Telegram
+TOKEN='token your telegram bot'
+ADMIN_CHAT_ID='telegram chat id where is there your bot'
+
+# Trello
+TRELLO_USER_API_KEY='trello api key from https://trello.com/app-key'
+TRELLO_USER_TOKEN='trello token from https://trello.com/app-key'
+TRELLO_ORG_NAME='short name of your org that'
```

```suggestion
TRELLO_ORG_NAME='short name of your workspace'
```

насколько я понял - "организация" в трелло - это воркспейс Оо

---

ага

---

они в апишке просто организациями называются, видимо щас поребрендили